README.md

```diff
@@ -143,6 +144,61 @@ kustomize build --enable-alpha-plugins github.com/mgoltzsche/khelm/example/cert-
 ```
 _When using kustomize 3 the option is called `--enable_alpha_plugins`._
 
+### kustomize containerized KRM function
+
+khelm can be used as a [kustomize](https://github.com/kubernetes-sigs/kustomize) [exec plugin](https://kubectl.docs.kubernetes.io/guides/extending_kustomize/containerized_krm_functions/).
+Though plugin support in kustomize is still an alpha feature, this form of
+extension seems destined to be graduated out of alpha (see [KEP
+2953](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/2953-kustomize-plugin-graduation)).
```

Comment:
One sentence per markdown line please :wink: - see [here](https://mattwidmann.net/notes/using-markdown-effectively/).

(This also applies to other paragraphs. Though in doubt I can make those changes afterwards.)

Reply:
Sorry I have a plugin in vscode that auto-wraps at 80 chars for other standards and I forget to check if it's appropriate sometimes... 😛